ui/packages/catalog-ui-search/src/main/webapp/component/visualization/inspector/inspector.view.js

```diff
@@ -13,16 +13,46 @@
  *
  **/
 /*global require*/
-const template = require('./inspector.hbs')
+import * as React from 'react'
+import styled from '../../../react-component/styles/styled-components'
+
 const Marionette = require('marionette')
 const CustomElements = require('../../../js/CustomElements.js')
 const MetacardView = require('../../tabs/metacard/tabs-metacard.view.js')
 const MetacardsView = require('../../tabs/metacards/tabs-metacards.view.js')
 const MetacardTitleView = require('../../metacard-title/metacard-title.view.js')
 
+const Empty = styled.div`
+  text-align: center;
+  display: none;
+  padding: 20px;
```

Comment:
If display is none, can't the `text-align` and `padding` attributes be removed from it?

Reply:
When the `.is_empty` class is added the display changes to block. I tried to move that logic to React but I would have had to migrate even more child components.